### ISSUE TITLE
Request: Add a License file to the project

### ISSUE THREAD

**Issue by slli** [twilio/twilio-chat-ios#27]
Due to company's policy, we can't use this library without a license file.

**Comment by berkus** [comment]
Good point, thanks!

**Comment by Gray-Wind** [comment]
@slli Hello, we are distributing SDKs under [TWILIO TOS](https://www.twilio.com/legal/tos). I will add a link to terms into the readme file.

**Comment by slli** [comment]
thanks a lot @Gray-Wind ! Would it be possible to also update the readme in [twilio-voice-ios](https://github.com/twilio/twilio-voice-ios) & [twilio-video-ios](https://github.com/twilio/twilio-video-ios) ?

**Comment by berkus** [comment]
@slli yep, I see you opened the tickets there already, people responsible for those repos should be able to help.

**Comment by Gray-Wind** [comment]
@slli I've pinged them, I expect them to add a license info soon. FYI, it would be the same license, `TWILIO-TOS`.
